internal/host/transition.go

```diff
@@ -544,6 +540,14 @@ func (th *transitionHandler) IsDay2Host(sw stateswitch.StateSwitch, args statesw
 	return hostutil.IsDay2Host(sHost.host), nil
 }
 
+func (th *transitionHandler) HostIsNotResponsive(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) (bool, error) {
```

Comment:
it will not work and break everything, it host was already rebooted you will get to this point and break all the installations. 
You will need the code that you deleted from https://github.com/openshift/assisted-service/pull/985/files#diff-0fa87e5c61dbb8ce3975c21b272167ca85a1a6538288ac08309b44ae153594a0L528-L531
Anyway please don't push it before you run test-infra on it.